app-former-deployment/src/test/java/org/kie/appformer/deployment/tests/RestPipelineImplTest.java

```diff
@@ -272,7 +274,7 @@ public Properties getProperties() {
                 put("create-repo", "true");
                 put("branch", "master");
                 put("out-dir", tempPath.getAbsolutePath());
-                put("origin", "https://github.com/mbarkley/livespark-playground");
+                put("origin", "https://github.com/mbarkley/app-former-playground");
```

Comment:
Thanks Michael :) @mcimbora see above.

I have a solution for everything, I think. I 'll explain in a jira.